.cz-config.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  types: [
+    {
+      value: 'chore',
```

Comment:
Types of commits to generate a commit message with  `yarn commit`. Example:

`feat: commit message`